php/team's-goals.php

```diff
@@ -0,0 +1,33 @@
+<?php
+///////////////////////////////////////////////////////////
+/// Receive team's id and returns an array with the goal's
+/// goals. Each array element have:
+///
+/// id: goal's id
+/// name: goal's name
+///
+/// Verify that goal belongs to the user before to get
+/// goals. If goal doesn't, it must an error message.
+//////////////////////////////////////////////////////////
+	require 'connect-db.inc';
+	session_start();
+
+	$db = connect_db();
+	$team_id = $_POST ['team_id'];
+
+	$stmt = $db->prepare('SELECT id, name FROM goal WHERE team_id = ?');
```

Comment:
Before you get team' goals, you must verify that current user is member of the team. If it's not member, returns an error message and the script ends